drivers/net/ipq6018/ipq6018_edma.c

```diff
@@ -1169,6 +1171,10 @@ static int ipq6018_eth_init(struct eth_device *eth_dev, bd_t *this)
 	}
 
 	if (linkup <= 0) {
+		if (link_retries-- > 0) {
+			mdelay(1000);
```

Comment:
maybe we can set link_retries to 100 and wait for 100 ms instead of 1000?

Reply:
> maybe we can set link_retries to 100 and wait for 100 ms instead of 1000?

That would throw lots of lines to the console. Every retry prints the status of all 5 PHYs, one per line. With 100ms delay you get 10 times the number of lines.

What would be the advantage of this?